incubator/catalog-schema-registry.spec/src/main/scripts/io/aklivity/zilla/specs/catalog/schema/registry/schema/schema.registry.schema.patch.json

```diff
@@ -39,6 +39,11 @@
                             {
                                 "type": "string",
                                 "default": "default"
+                            },
+                            "cacheTTL":
```

Comment:
Let's rename this to `max-age` and parse it as a `Duration` same as `max-age` in `http` binding options and `challenge` in `jwt` guard options.

See https://github.com/aklivity/zilla/blob/develop/runtime/binding-http/src/main/java/io/aklivity/zilla/runtime/binding/http/internal/config/HttpOptionsConfigAdapter.java#L359 as an example.